### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by tklecka** [OpenDroneAT/OpenDrone#94]
Do it 😒

**Comment by gunjanmimo** [comment]
I want to do this, may you help me?

**Comment by tklecka** [comment]
> I want to do this, may you help me?

Of course, we'd be happy to help. @MarkusKurzmann is responsible for the Wiki.

**Comment by MarkusKurzmann** [comment]
@gunjanmimo 
I appreciate that you want to help me with the wiki! 👍 
I just changed the part of the installation a little bit so that it runs on a fresh version of raspbian buster lite. If you want (and if you have a raspberry pi) you can try out the installation and give me some feedback if there were any problems with the installation for you.

Kind regards,
Markus Kurzmann
